application/classes/Ushahidi/Formatter/Post/GeoJSONCollection.php

```diff
@@ -47,6 +47,20 @@ public function __invoke($entities)
 
 			if (! empty($geometries))
 			{
+				$set_name = [];
+				$set_ids = $entity->sets;
+
+				if ($set_ids != null) {
+					$query = DB::select('sets.name')
```

Comment:
I think this needs to differentiate between collections and saved searches. Both use the sets table in the backend - but the public api always references `collections` or `savedsearches`. Could you add them as 2 separate fields?